tests/orquestra/quantum/circuits/_gates_test.py

```diff
@@ -282,7 +283,9 @@ def test_dagger_of_power_gate_power_gate_of_dagger(self, gate, exponent):
             assert power_gate.dagger == gate.dagger.power(exponent)
 
     def test_creating_power_gate_from_power_gate(self, gate, exponent):
-        if len(gate.free_symbols) == 0:
+        # We exclude ECR ** 0.5 gate from this test because it would require computing
+        # ECR ** 0.25, which currently sympy (as of version 1.9) cannot handle.
+        if len(gate.free_symbols) == 0 and (exponent != 0.5 or gate.name != "ECR"):
```

Comment:
If that's the case I would actually add an `pytest.xfail` for this case. 
Though it's a little bit weird that it doesn't work for this particular  gate, could you provide some extra details?